sigh/lib/sigh/resign.rb

```diff
@@ -60,8 +73,11 @@ def resign(ipa, signing_identity, provisioning_profiles, entitlements, version,
 
       puts(command.magenta)
       puts(`#{command}`)
+      command_return_value = $?.to_i
+
+      `security list-keychains -s #{previous_keychains}` if previous_keychains
```

Comment:
Can this get executed inside of an `ensure` at the bottom of this method? Things _look_ save enough where an error/exception will never get thrown but it seems like this is a pretty important step that we **always** want to get executed no matter what. I think? 🤔

Reply:
Yes, you are right. It does make sense to put this into an ensure block (Although its usually not to much of a problem for the user if an extra keychain added). See my newest commit. It now has the ensure block added.